core/math/quaternion.cpp

```diff
@@ -292,6 +292,19 @@ real_t Quaternion::get_angle() const {
 	return 2 * Math::acos(w);
 }
 
+Quaternion Quaternion::get_twist(const Vector3 &v) const
+{
+#ifdef MATH_CHECKS
+	ERR_FAIL_COND_V_MSG(!is_normalized(), Quaternion(), "The start quaternion must be normalized.");
```

Comment:
```suggestion
	ERR_FAIL_COND_V_MSG(!is_normalized(), Quaternion(), "The quaternion must be normalized.");
```
No "start" here, also does this actually work with a non-normalized axis? It being an axis implies it's normalized

Reply:
I realized I never tested with non-normalized axis, since I mostly use it with the Y axis.
It give a rotation around the correct axis, but value seems multiplied with the magnitude of the vector. Interesting, but I don't know if it has a use case.

I'll add a check for the axis vector to be normalized.